arch/arm/mach-imx/video.c

```diff
@@ -32,8 +37,14 @@ int board_video_skip(void)
 	}
 
 	if (i < display_count) {
+#if defined(CONFIG_VIDEO_IPUV3)
 		ret = ipuv3_fb_init(&displays[i].mode, displays[i].di ? 1 : 0,
 				    displays[i].pixfmt);
+#elif defined(CONFIG_VIDEO_MXS)
+		ret = mxs_lcd_panel_setup(displays[i].mode,
+					displays[i].pixfmt,
+				    displays[i].bus);
```

Comment:
U-Boot patch checker: Alignment should match open parenthesis.
```suggestion
					  displays[i].bus);
```